src/translation/translation_de_DE.ts

```diff
@@ -2656,109 +2660,112 @@ Wir haben Deinen Kanal stummgeschaltet und die Funktion &apos;Stummschalten&apos
     <message>
-        <location filename="../clientsettingsdlgbase.ui" line="227"/>
+        <location filename="../clientsettingsdlgbase.ui" line="250"/>
         <source>Meter Style</source>
         <translation>Pegel-Messer-Stil</translation>
     </message>
     <message>
-        <location filename="../clientsettingsdlgbase.ui" line="241"/>
+        <location filename="../clientsettingsdlgbase.ui" line="289"/>
         <source>Mixer Rows</source>
         <translation>Mischpult-Reihen</translation>
     </message>
     <message>
-        <location filename="../clientsettingsdlgbase.ui" line="248"/>
+        <location filename="../clientsettingsdlgbase.ui" line="321"/>
         <source>Audio Alerts</source>
         <translation>Audiohinweise</translation>
     </message>
     <message>
-        <location filename="../clientsettingsdlgbase.ui" line="339"/>
+        <location filename="../clientsettingsdlgbase.ui" line="362"/>
         <source>Audio/Network Setup</source>
         <translation>Audio-/Netzwerk-Einstellungen</translation>
     </message>
     <message>
-        <location filename="../clientsettingsdlgbase.ui" line="349"/>
+        <location filename="../clientsettingsdlgbase.ui" line="375"/>
         <source>Audio Device</source>
         <translation>Audiogerät</translation>
     </message>
     <message>
-        <location filename="../clientsettingsdlgbase.ui" line="690"/>
+        <location filename="../clientsettingsdlgbase.ui" line="737"/>
         <source>Jitter Buffer</source>
         <translation>Netzwerkpuffer</translation>
     </message>
     <message>
-        <location filename="../clientsettingsdlgbase.ui" line="702"/>
+        <location filename="../clientsettingsdlgbase.ui" line="749"/>
         <source>Auto</source>
         <translation></translation>
     </message>
     <message>
-        <location filename="../clientsettingsdlgbase.ui" line="711"/>
+        <location filename="../clientsettingsdlgbase.ui" line="758"/>
         <source>Local</source>
         <translation>Lokal</translation>
     </message>
     <message>
-        <location filename="../clientsettingsdlgbase.ui" line="724"/>
+        <location filename="../clientsettingsdlgbase.ui" line="771"/>
         <source>Server</source>
         <translation>Server</translation>
     </message>
     <message>
-        <location filename="../clientsettingsdlgbase.ui" line="741"/>
-        <location filename="../clientsettingsdlgbase.ui" line="754"/>
+        <location filename="../clientsettingsdlgbase.ui" line="788"/>
+        <location filename="../clientsettingsdlgbase.ui" line="801"/>
         <source>Size</source>
         <translation>Größe</translation>
     </message>
     <message>
-        <location filename="../clientsettingsdlgbase.ui" line="883"/>
+        <location filename="../clientsettingsdlgbase.ui" line="899"/>
+        <source>Small Network Buffers</source>
+        <translation type="unfinished"></translation>
+    </message>
```

Comment:
This is one example (there are more) for translation-relevant changes -- strings moved from .cpp to .ui and are therefore lost.
If possible (and the context is unchanged), it might make sense to move those manually as part of this PR as well?

If we want this to go into 3.9.1, which we planned to be without new translations for now, that would be a must.

Reply:
What I've now done is made sure the ts files get the ui and cpp updates as well, but flagged as unfinished.